bin/v-import-cpanel

```diff
@@ -251,6 +251,27 @@ rsync -av --exclude-from='exclude_path' homedir/public_html/ /home/$new_user/web
 chown $new_user:$new_user -R /home/$new_user/web/$main_domain1/public_html
 chmod 751 /home/$new_user/web/$main_domain1/public_html
 
+####### Set Domain PHP version
+PHP_VERSION_LINE=$(grep -r "phpversion:" userdata/$main_domain1)
+CPANEL_PHP_VERSION=${PHP_VERSION_LINE#*: }
+CPANEL_PHP_VERSION=$(echo $CPANEL_PHP_VERSION | grep -oP '(?<=php)\d+')
+HESTIA_PHP_VERSION="PHP-${CPANEL_PHP_VERSION:0:1}_${CPANEL_PHP_VERSION:1}"
+
+if /usr/local/hestia/bin/v-list-web-templates-backend | grep -qw "$HESTIA_PHP_VERSION"; then
```

Comment:
Please use:
$BIN

instead of /usr/local/hestia/bin

Reply:
Done, Thank you